checkers/forbidden_import_checker.py

```diff
@@ -8,22 +8,30 @@
 class ForbiddenImportChecker(BaseChecker):
     __implements__ = IAstroidChecker
 
-    name = 'custom'
+    name = 'forbidden import'
     msgs = {'E9999':
                 ('You may not import any modules - you imported \033[4;34m%s\033[0m on line %s.',
                  'forbidden-import',
                  'Used when you use import')}
-    options = ()
+    options = (('allowed_import_modules',
+                {'default': (),
+                 'type': 'csv',
+                 'metavar': '<modules>',
+                 'help': 'Allowed modules to be imported.'}
+               ),)
+
     # this is important so that your checker is executed before others
     priority = -1
 
     @check_messages("forbidden-import")
     def visit_import(self, node):
         """visit and Import node"""
-        self.add_message('forbidden-import',
-                         node=node,
-                         args=(', '.join(map(lambda x: x[0], node.names)),
-                               node.lineno))
+        temp = []
```

Comment:
You can use a list comprehension to define this as well. (Remember that you can use `if`s in list comprehensions too.